### DISCUSSION TITLE
Why is the method unsafeUpdateConfig() named unsafe?

### DISCUSSION THREAD

**Discussion by HenrikWendt** [cashapp/paparazzi#944] [Q&A] [answered]
Hi!

We are a team of developers working with Paparazzi. In our case we are woking with different themes that needs to be applied to different components. Due to some cashing problems some components do keep the first theme set and therefore we need a way to re-apply the next theme. 

We found the method unsafeUpdateConfig() and it does the job for us, but we are a bit worried about the method name, what makes it unsafe?

Kind regards,

Henrik

**Comment by ko-bra** [comment]
@luis-cortes Could you (as the creator of this method) please provide some guidance here?
Much appreciated! 🙏

**Comment by luis-cortes** [comment, marked as answer]
It's unsafe because you still have to reinflate your view to see any changes you've made to the device config.

See #428 for more details.

**Reply by HenrikWendt** [reply]
Thank you for clarifying! ✅
